src/main/kotlin/org/jitsi/nlj/rtp/TransportCcEngine.kt

```diff
@@ -125,6 +130,13 @@ class TransportCcEngine(
                 }
             }
         }
+        if (missingPacketDetailSeqNums.isNotEmpty()) {
+            logger.warn("TCC packet contained sequence numbers:\n" +
```

Comment:
Another quibble: I don't like having newlines in log messages, it makes it hard to grep log files.

Reply:
yeah i didn't have this originally, but since they usually come in such big bursts i thought it'd be difficult to parse (mentally) on one line.  this is a good point, though, i can change it back